internal/cli/root.go

```diff
@@ -8,6 +8,8 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var Version = "0.1.0"
```

Comment:
I would leave "valid" semver to actual tagged version. Let's keep a generic `0.0.0-dev` as default

```suggestion
var Version = "0.0.0-dev"
```